scripts/run_integration.sh

```diff
@@ -141,38 +141,62 @@ function expand_bootlin_config() {
   local -r GCC_DIR=${ARCHIVE_DIR}/${GCC_RELATIVE_DIR}
 
   case "${TARGET}" in
-    "aarch64")
-      local -r TOOLCHAIN_URL="https://toolchains.bootlin.com/downloads/releases/toolchains/aarch64/tarballs/aarch64--glibc--stable-2021.11-1.tar.bz2"
+    "arm" | "armv7-eabihf")
+      local -r TOOLCHAIN_URL="https://toolchains.bootlin.com/downloads/releases/toolchains/armv7-eabihf/tarballs/armv7-eabihf--glibc--stable-2024.05-1.tar.xz"
```

Comment:
[nit] Maybe use a function to create the url?

Reply:
at least a macro for the URL prefix, on my way...

Reply:
SGTM I don't have a strong opinion on this, copy / paste is OK.

Reply:
see offlline: Better to keep full url since it'd be less error prone to copy/paste them from "copy link address" from 
https://toolchains.bootlin.com